tests/variable/variable.cpp

```diff
@@ -134,6 +151,34 @@ TEST(Variable, addTwoVariables) {
     EXPECT_EQ(e, a);
 }
 
+TEST(Variable, addVariables) {
+	const std::string e =
+		"<component>"
+		"<variable name=\"var1\" units=\"dimensionless\"/>"
+		"<variable name=\"var2\"/>"
+		"<variable name=\"var3\"/>"
```

Comment:
Good idea, @jonc125 , as a way to pick up more permutations in a single test. This bugfix pull actually pre-dates the addition of initial values so will add those in under that pull set. :-)